tests/acceptance/test_update.py

```diff
@@ -142,7 +142,7 @@ def test_broken_image_update(self, bitbake_variables):
             subprocess.call("dd if=/dev/zero of=image.dat bs=1M count=0 seek=16", shell=True)
             subprocess.call("mender-artifact write rootfs-image -t %s -n test-update -u image.dat -o image.mender" % image_type, shell=True)
             put("image.mender", remote_path="/var/tmp/image.mender")
-            run("mender -rootfs /var/tmp/image.mender")
+            run("mender -rootfs /var/tmp/image.mender -f")
```

Comment:
I'm curious why this is needed?

Reply:
If your build has state scripts in it, then the command line installation will fail.  My normal config adds some so I had to add this to run the tests locally.